spectral_cube/io/fits.py

```diff
@@ -44,7 +44,7 @@ def is_fits(input, **kwargs):
         return False
 
 
-def read_data_fits(input, hdu=None, **kwargs):
+def read_data_fits(input, hdu=None, mode='denywrite', **kwargs):
```

Comment:
Yes.  But I think this should be the default, as we don't expect users to be modifying the cube data inplace.